src/util/callApi.js

```diff
@@ -8,12 +8,13 @@ function sanitizeUri(endpoint) {
  * Inspect the response.status number within the resolved callback to add conditional handling of server errors to your code.
  */
 export default function callApi({
+  api_endpoint = process.env.REACT_APP_API_ENDPOINT,
   uri,
   method = "GET",
   body = undefined,
   headers
 }) {
-  let endpoint = process.env.REACT_APP_API_ENDPOINT + sanitizeUri(uri);
+  let endpoint = api_endpoint + sanitizeUri(uri);
```

Comment:
why do we need to pass api_endpoint each time? What is the disadvantage of reading base url from env variable and then just appending URI part?

Reply:
Actually, my intention was to make use of callApi with different api_endpoints. I thought it would be legit since there is a default value for the api_endpoint parameter.

Reply:
Okay, actually that's a good idea. So can we simplify the other usages and calls by removing the argument, cus it sends the same default value, and other api calls are going to be mostly using that default value.

Reply:
Removed endpoint parameter from callApi. As you suggested, there should be other methods that call different endpoints since the callApi is targeted for the backend.